walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +275,146 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
-}
 
+  @Test
+  public void fullyBookedOptionalAttendee() {
+    // Have the optional attendee be fully booked. We should see three options because the algorithm
+    // ignores them
+    //
+    // Events  :       |--A--|     |--B--|
+    //           |-------------*C--------------|
```

Comment:
nit: specify in the text that you have three attendees, A, B, and C, where C is optional. It took me a moment to understand your diagram -- though it's very clear and useful once I did understand!

You might also re-label the diagram as "Input" and "Output" rather than "Events" and "Options". But best to be consistent with the rest of the file, and it's probably not worth rewriting every comment.